string env_stl_path
string[] ob_names
string[] ob_stl_paths
bool is_cam
float64 forced_z
float64[] min_limits
float64[] max_limits
geometry_msgs/Point[] env_polygon
int32 limit_collision
bool reset
---
float64[] x
float64[] y
float64[] z
float64[] yaw